docs/team/tianyong-goh.md

```diff
@@ -0,0 +1,73 @@
+---
+layout: page
+title: Goh Tian Yong's Project Portfolio Page
+---
+
+## Project: WishfulShrinking
+
+WishfulShrinking is a desktop diet manager refactor from an application called [AddressBook Level 3](https://github.com/nus-cs2103-AY2021S1/tp).
+ It is an application used to helps on **managing the user's on-hand ingredients, organise personal recipes and track user's diet**.
+ Wishful Shrinking facilitates a **healthier diet** in three main ways: 
+
+1. Provide a **source of healthy, customizable recipes** 
+1. **Recommend recipes** to improve ease of home cooking 
+1. **Track daily food and calorie** intake<br><br>
+
+Wishful Shrinking targets **office workers** who tend to discount healthy eating. Office workers are also more
+ familiar with desktop applications and typing and correspondingly, Wishful Shrinking optimized for fast and efficient
+ typers as it uses a Command Line Interface (CLI) and Graphical User Interface (GUI) created with JavaFX.
+ Wishful Shrinking is available for the Linux, Unix, Windows and Mac OS operating systems.
+ It is written in Java, and has about 10 kLoC. 
+ 
+
+Given below are my contributions to the project.
+
+* **Code contributed**: [RepoSense link](https://nus-cs2103-ay2021s1.github.io/tp-dashboard/#breakdown=true&search=tianyong-goh&sort=groupTitle&sortWithin=title&since=2020-08-14&timeframe=commit&mergegroup=&groupSelect=groupByRepos&checkedFileTypes=docs~functional-code~test-code~other)
+
+* **New Feature**: Added the `Eat Recipe` command. (Pull Requests [\#51](https://github.com/AY2021S1-CS2103T-W10-2/tp/pull/51))
+  * What it does: allows the user "eat" the recipe that is listed in the recipes' collection.
+  * Justification: This feature improves the product as it is the base feature of Wishful Shrinking to help user to keep track of their daily consumption.
+  * Highlights: This feature work similar to `add Recipe`/ `add Ingredient` feature.
+   However, it instead intakes a positive index as an argument to indicate the specific recipe,
+   get the recipe's data from Recipe List and add the recipe's data to the Consumption List.
+  * Credits: AddressBook Level 3 `delete` feature.
```

Comment:
delete? not add?

Reply:
ya is delete cause eatR use INDEX. I might change the add Recipe and add Ingredient to delete cause now I feel It actually work more like a delete but instead of removing, it adds.